keyboards/splitkb/halcyon/ferris/keymaps/default/keymap.json

```diff
@@ -0,0 +1,68 @@
+            "KC_LCBR"     , "KC_RCBR"     , "KC_LPRN"        , "KC_RPRN"       , "KC_AT"         , "KC_TRNS"       , "KC_NO"       , "KC_EQL"         , "KC_PLUS"       , "KC_PERC",
+            "KC_TRNS"     , "KC_EXLM"     , "KC_LBRC"        , "KC_RBRC"       , "KC_TRNS"       , "KC_TRNS"       , "KC_TRNS"     , "KC_TRNS"        , "KC_TRNS"       , "KC_TRNS",
+                                                               "KC_VOLD"       , "KC_TRNS"       , "KC_TRNS"       , "KC_VOLU"
+        ],
+        [
+            "KC_TRNS"     , "KC_TRNS"     , "KC_TRNS"        , "KC_TRNS"       , "KC_TRNS"       , "KC_TRNS"       , "KC_F7"       , "KC_F8"          , "KC_F9"         , "KC_F10",
+            "KC_TRNS"     , "KC_NO"       , "LCTL(KC_LALT)"  , "KC_TRNS"       , "KC_TRNS"       , "KC_TRNS"       , "KC_F4"       , "KC_F5"          , "KC_F6"         , "KC_F11",
+            "KC_TRNS"     , "KC_TRNS"     , "KC_TRNS"        , "KC_TRNS"       , "KC_TRNS"       , "KC_TRNS"       , "KC_F1"       , "KC_F2"          , "KC_F3"         , "KC_F12",
+                                                               "KC_TRNS"       , "KC_TRNS"       , "KC_TRNS"       , "KC_TRNS"
+        ],
+        [
+            "KC_PSLS"     , "KC_7"        , "KC_8"           , "KC_9"          , "KC_PPLS"       , "KC_TRNS"       , "KC_TRNS"     , "KC_TRNS"        , "KC_TRNS"       , "KC_TRNS",
+            "KC_0"        , "KC_1"        , "KC_2"           , "KC_3"          , "KC_PMNS"       , "KC_TRNS"       , "KC_TRNS"     , "KC_TRNS"        , "KC_NO"         , "KC_TRNS",
+            "KC_PAST"     , "KC_4"        , "KC_5"           , "KC_6"          , "KC_PEQL"       , "KC_TRNS"       , "KC_TRNS"     , "KC_TRNS"        , "KC_TRNS"       , "KC_TRNS",
+                                                               "KC_TRNS"       , "KC_TRNS"       , "KC_TRNS"       , "KC_TRNS"
+        ],
+        [
+            "KC_TRNS"     , "KC_TRNS"     , "KC_COLN"        , "KC_ESC"        , "KC_TRNS"       , "KC_TRNS"       , "KC_TRNS"     , "KC_TRNS"        , "KC_TRNS"       , "KC_DEL",
+            "KC_TRNS"     , "KC_PERC"     , "KC_SLSH"        , "KC_ENT"        , "KC_TRNS"       , "DF(1)"         , "KC_LGUI"     , "KC_TRNS"        , "KC_TRNS"       , "KC_TRNS",
+            "KC_TRNS"     , "KC_TRNS"     , "KC_TRNS"        , "KC_EXLM"       , "KC_TRNS"       , "DF(0)"         , "KC_TRNS"     , "RALT_T(KC_COMM)", "RCTL_T(KC_DOT)", "QK_BOOT",
+                                                               "KC_TRNS"       , "KC_TAB"        , "KC_NO"         , "KC_TRNS"
+        ]
+    ],
+    "mouse_key": {
+        "enabled": true,
+        "interval": 16,
+        "time_to_max": 40,
+        "delay": 100,
+        "wheel_delay": 100
+    },
+    "tapping": {
+        "term": 200,
+        "permissive_hold": true,
+        "force_hold": true
+    }
```

Comment:
These are not valid top level keys, where they would be required to be within a parent `config` object to actually function